src/SPC/doctor/item/MacOSToolCheckList.php

```diff
@@ -40,7 +40,7 @@ public function checkBrew(): ?CheckResult
         if (($path = $this->findCommand('brew')) === null) {
             return CheckResult::fail('Homebrew is not installed', 'brew');
         }
-        if ($path !== '/opt/homebrew/bin/brew' && php_uname('m') === 'arm64') {
+        if ($path !== '/opt/homebrew/bin/brew' && getenv('SPC_ARCH') === 'arm64') {
```

Comment:
didn't you rename it to aarch64 earlier?

Reply:
Oh, actually I should use `GNU_ARCH` here. See my comments below.

Reply:
I think you should check MAC_ARCH against aarch64 here, given that this is the MacOSToolCheckList.